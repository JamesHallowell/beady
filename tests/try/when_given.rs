use beady::scenario;

#[scenario]
fn when_given_is_invalid() {
    'given_something: {
        'when_something: {
            'given_something: {
                assert!(true);
            }
        }
    }
}

fn main() {}
