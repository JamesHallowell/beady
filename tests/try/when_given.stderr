error: Only "then" or "and_when" are allowed inside "when" section
 --> tests/try/when_given.rs:8:13
  |
8 |             'given_something: {
  |             ^^^^^^^^^^^^^^^^
